SmthGrey/ReadMe.txt

```diff
@@ -0,0 +1,16 @@
+SmthGrey
+Smoothswing sound font for ProffieOS by Thexter.
+
+
+Modifications by Brian Conner 2022:
+Any additional sounds sourced from creative-commons licensed freesound.org.
+
+Hierarchy made to be FAT32 friendly.
+Added boot.wav.
```

Comment:
These changes should be a part of the PR that submitted them, not batched together into one "txt file update".

Reply:
I may have misinterpreted your request :

>do all the renames in one PR, then fix config files, and then any add/delete/modify separately